src/test-e2e/marketplace/marketplace.po.ts

```diff
@@ -0,0 +1,26 @@
+import { protractor, ElementArrayFinder, ElementFinder } from 'protractor';
+import { E2EHelpers } from '../helpers/e2e-helpers';
+import { browser, promise } from 'protractor';
+import { Page } from '../po/page.po';
+import { ListComponent } from '../po/list.po';
+
+export class MarketplacePage extends Page {
+  helpers = new E2EHelpers();
+
+  servicesList = new ListComponent();
+
+  navigateTo() {
```

Comment:
If you add a constructor that sets the URL to /marketplace, you don't need to override this method.